packages/aws-rfdk/lib/deadline/scripts/bash/installDeadlineRepository.sh

```diff
@@ -18,6 +18,8 @@ Required arguments:
 Optional arguments
   -s Deadline Repository settings file to import.
   -o The UID[:GID] that this script will chown the Repository files for. If GID is not specified, it defults to be the same as UID."
+  -c Secret management admin credentials ARN. If this parameter is specified, secrets management will be enabled.
+  -r Region where stacks are deployed. Required to get secret management credentials.
 
 while getopts "i:p:v:s:o:" opt; do
```

Comment:
```suggestion
while getopts "i:p:v:s:o:c:r:" opt; do
```

I missed this last time. This needs to be updated for these args to be captured